GANDLF/compute/forward_pass.py

```diff
@@ -341,7 +341,7 @@ def validate_network(
                     if image.shape[-1] > 1:
                         result_image = sitk.GetImageFromArray(pred_mask)
                     else:
-                        result_image = sitk.GetImageFromArray(pred_mask.squeeze(0))
+                        result_image = sitk.GetImageFromArray(pred_mask.squeeze(-1))
```

Comment:
I think this is coming up because the `pred_mask` is becoming `[1,X,Y]` instead of `[X,Y,1]` (because of how the image axes are defined). 

I would suggest replace L341-L344 with the following code (just copy-paste, it is already linted):

```python
                    pred_mask = (
                        pred_mask.squeeze(0)
                        if pred_mask.shape[0] == 1
                        else (
                            pred_mask.squeeze(-1)
                            if pred_mask.shape[-1] == 1
                            else pred_mask
                        )
                    )
                    result_image = sitk.GetImageFromArray(pred_mask)
```